ts/src/hitbtc.ts

```diff
@@ -1,14 +1,10 @@
-
```

Comment:
@samgermain if this is a rename only why do we have two different files and so many changes?

Reply:
we already had hitbtc and hitbtc3. 

This PR moves all the code on the file hitbtc3.ts to hitbtc.ts, which removes all the code that was originally inside hitbtc.ts. Also all the code from hitbtc3.ts was removed and replaced with the few lines of code to make an alias exchange

Reply:
yes, Sam's edit of htbtc3 seems correct